python/port/mod/turtle/turtle.cpp

```diff
@@ -74,19 +74,22 @@ void Turtle::left(mp_float_t angle) {
 void Turtle::circle(mp_int_t radius, mp_float_t angle) {
   mp_float_t oldHeading = heading();
   mp_float_t length = std::fabs(angle * k_headingScale * radius);
+  int direction = 1; // 1=fwd, -1=backward
```

Comment:
Just easier to read, but it's a detail
```suggestion
  int direction = 1; // 1=forward, -1=backward
```